src/sqlast/mod.rs

```diff
@@ -69,55 +69,55 @@ pub enum ASTNode {
     /// Multi-part identifier, e.g. `table_alias.column` or `schema.table.col`
     SQLCompoundIdentifier(Vec<SQLIdent>),
     /// `IS NULL` expression
-    SQLIsNull(Box<ASTNode>),
+    SQLIsNull(Box<Expr>),
     /// `IS NOT NULL` expression
-    SQLIsNotNull(Box<ASTNode>),
+    SQLIsNotNull(Box<Expr>),
     /// `[ NOT ] IN (val1, val2, ...)`
     SQLInList {
-        expr: Box<ASTNode>,
-        list: Vec<ASTNode>,
+        expr: Box<Expr>,
+        list: Vec<Expr>,
         negated: bool,
     },
     /// `[ NOT ] IN (SELECT ...)`
     SQLInSubquery {
-        expr: Box<ASTNode>,
+        expr: Box<Expr>,
         subquery: Box<SQLQuery>,
         negated: bool,
     },
     /// `<expr> [ NOT ] BETWEEN <low> AND <high>`
     SQLBetween {
-        expr: Box<ASTNode>,
+        expr: Box<Expr>,
         negated: bool,
-        low: Box<ASTNode>,
-        high: Box<ASTNode>,
+        low: Box<Expr>,
+        high: Box<Expr>,
     },
     /// Binary operation e.g. `1 + 1` or `foo > bar`
     SQLBinaryOp {
-        left: Box<ASTNode>,
+        left: Box<Expr>,
         op: SQLBinaryOperator,
-        right: Box<ASTNode>,
+        right: Box<Expr>,
     },
     /// Unary operation e.g. `NOT foo`
     SQLUnaryOp {
         op: SQLUnaryOperator,
-        expr: Box<ASTNode>,
+        expr: Box<Expr>,
     },
     /// CAST an expression to a different data type e.g. `CAST(foo AS VARCHAR(123))`
     SQLCast {
```

Comment:
My version of `cargo fmt` reformats this on one line. I wonder why CI doesn't catch it..

Reply:
What version of the Rust toolchain are you using? Perhaps the formatting here is different in different versions.

Reply:
Never mind. I think something's broken in CI.